holmes/plugins/toolsets/bash/kubectl/kubectl_events.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import re
+from typing import Any
+
+from holmes.plugins.toolsets.bash.common.stringify import escape_shell_args
+from holmes.plugins.toolsets.bash.common.validators import regex_validator
+
+
+def create_kubectl_events_parser(kubectl_parser: Any):
+    parser = kubectl_parser.add_parser(
+        "events",
+        help="List events",
+        exit_on_error=False,  # Important for library use
+    )
+    parser.add_argument(
+        "-n",
+        "--namespace",
+        type=regex_validator("namespace", re.compile(r"^[a-z0-9][a-z0-9\-]*$")),
+    )
+    parser.add_argument("-A", "--all-namespaces", action="store_true")
+    parser.add_argument(
+        "-l",
+        "--selector",
+        type=regex_validator("selector", re.compile(r"^[a-zA-Z0-9\-_.=,!()]+$")),
+    )
+    parser.add_argument(
+        "--field-selector",
+        type=regex_validator("field selector", re.compile(r"^[a-zA-Z0-9\-_.=,!()]+$")),
+    )
+    parser.add_argument(
```

Comment:
Out of couriosity moving only the funciton name instead of lambda won't work? just trying to understand why we need the lambda here.